### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by celoazevedo**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+    expect(renderedHeader.length).toEqual(1)
+    expect(renderedFooter.length).toEqual(1)
+  })
+  it('provides a route"/" to the home component', ()=>{
+    // arrange
+    const renderedApp = shallow(<App />)
+    // act
+    const renderedHomeRoute = renderedApp.find('[path="/"]')
+    //  console.log("rendered home route debug", renderedHomeRoute.debug())
+    //  console.log("rendered home route props", renderedHomeRoute.props())
```
Comment (WalkerTexas-coder) [inline src/App.test.js:27]: If you're done with these console logs lets delete them.

File: src/components/Header.js [hunk 2]
```diff
@@ -1,19 +1,33 @@
-import React, { Component } from 'react';
-import Logo from '../assets/turtle_header.jpeg'
-class Header extends Component {
-    render() {
+// import React, { Component } from 'react';
+import React, { useState } from 'react';
+import { Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, NavItem, NavLink } from 'reactstrap';
+// import Logo from '../assets/turtle_header.jpeg'
```
Comment (WalkerTexas-coder) [inline src/components/Header.js:4]: are you saving this for later implementation?

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
Looking good so far, I left a few comments about cleaning up console.logs. 

It looks like y'all are missing the Home.test.js file for test coverage of your Home Component. Add it and it's test coverage and I'll take another look.

File: src/App.test.js (hunk 1 above)
Comment (celoazevedo) [reply src/App.test.js:27]: can we keep them for future references? just for a little bit?

File: src/components/Header.js (hunk 2 above)
Comment (celoazevedo) [reply src/components/Header.js:4]: took care of these. we can add them later if we need them.

**Review by WalkerTexas-coder — APPROVED** [review]
Looks good!

**Pull request merged**